package.json

```diff
@@ -21,7 +21,8 @@
 		"docs:build": "cd docs && pnpm build",
 		"docs:preview": "cd docs && pnpm preview",
 		"ci:build": "pnpm build && pnpm docs:build",
-		"ci:test": "pnpm lint && pnpm test"
+		"ci:test": "pnpm lint && pnpm test",
+		"release": "pnpm changeset version && pnpm install && git add -A && git commit -m 'Version Packages' && changeset tag && pnpm publish -r"
```

Comment:
```suggestion
		"release": "pnpm install && pnpm changeset version && git add -A && git commit -m 'Version Packages' && pnpm changeset tag && pnpm publish -r"
```

Reply:
The install command has to happen after `changeset version` which is the root cause of the problems we are having. `pnpm` mirrors the package versions in all`package.json` files in their lockfile.

Reply:
Ohh, gotcha.

Never used pnpm, but that looked odd to me.